presto-main/src/main/java/com/facebook/presto/cost/ComparisonStatsCalculator.java

```diff
@@ -31,26 +36,31 @@
 
 public final class ComparisonStatsCalculator
 {
+    private static final Logger log = Logger.get(ComparisonStatsCalculator.class);
+
     private ComparisonStatsCalculator() {}
 
     public static PlanNodeStatsEstimate estimateExpressionToLiteralComparison(
             PlanNodeStatsEstimate inputStatistics,
             VariableStatsEstimate expressionStatistics,
             Optional<VariableReferenceExpression> expressionVariable,
             OptionalDouble literalValue,
-            ComparisonExpression.Operator operator)
+            ComparisonExpression.Operator operator,
+            Optional<Session> session)
```

Comment:
If only histogram field in the session is used, can we pass a boolean instead of the whole session around?

Reply:
Do you see a specific downside to passing the session? If we intend to make changes in the future which reference more feature flags, I feel it would be better to pass the session rather than adding a new argument for each potentially new flag

Reply:
I was thinking about keeping it consistent as I see in some other cases it's passing a boolean flag. But also fine if you think passing session is better.